wsclient/src/main/java/org/hpccsystems/ws/client/antlr/EclRecord.g4

```diff
@@ -117,7 +119,7 @@ CPAREN             : ')';
 OCURLY             : '{';
 CCURLY             : '}';
 COMMA              : ',';
-SEMI               : ';';
+SEMI               : (';' | WS '=>');
```

Comment:
This seems odd, are we treating the payload_sep equivalent to an end of line SEMI?

Reply:
It is odd, Antlr grammar is a PITA. This is the only way I was able to allow ecl field definitions that did NOT end in a ";", but ONLY when it was followed by a =>.